src/csvinputfileconn.cc

```diff
@@ -73,15 +73,8 @@ namespace dd
             else // in training mode, collect bounds, then scale in another
                  // pass over the data
               {
-                if (_cifc->_min_vals.empty() && _cifc->_max_vals.empty())
-                  _cifc->_min_vals = _cifc->_max_vals = vals;
-                for (size_t j = 0; j < vals.size(); j++)
-                  {
-                    _cifc->_min_vals.at(j)
-                        = std::min(vals.at(j), _cifc->_min_vals.at(j));
-                    _cifc->_max_vals.at(j)
-                        = std::max(vals.at(j), _cifc->_max_vals.at(j));
-                  }
+                throw InputConnectorBadParamException(
+                    "auto scaling while training from memory not implemented");
```

Comment:
indeed this (removed) code is completely buggy, min and max are updated, but values are never scaled, the second pass mentioned in comment is nowhere to be found. As training from memory is not a usual case, I prefered to remove this code entirely

Reply:
Agreed

Reply:
after some thoughts, I think data is well scaled in case of minmax, but not here, I will fix / clean things in another PR. I restored previous behavior

Reply:
If we remove from in memory, let's have an error raised when scaling is specified on an in-memory training job.

Reply:
indeed scaling when training from memory is now properly supported, both in minmax and znorm (and code is a bit cleaned up around this)